ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/view/service/DcatViewService.java

```diff
@@ -4,6 +4,7 @@
 import be.vlaanderen.informatievlaanderen.ldes.server.domain.viewcreation.valueobjects.ViewName;
 import org.apache.jena.rdf.model.Model;
 
+import java.util.List;
 import java.util.Optional;
 
 public interface DcatViewService {
```

Comment:
I know it can be a standard of having Services and Repositories.. but it looks a bit like useless to have a Service that does little more than just a pass through.

Reply:
It does some minor mapping, sometimes an existence check. 

Personally I don't mind this setup as it is consistent across different flows. You know where to look for what and have a place to add logic when required. It also improves maintainability as I believe the repository should have no logic at all (except for fetching the entities and mapping them to value objects).
Sometimes calling the repository and sometimes the service from different places and for different methods would be less streamlined.

You can add this for discussion on the dev meeting if you want.